src/items.rs

```diff
@@ -638,6 +638,46 @@ fn format_tuple_struct(context: &RewriteContext,
     Some(result)
 }
 
+pub fn rewrite_type_alias(context: &RewriteContext,
+                          indent: Indent,
+                          ident: ast::Ident,
+                          ty: &ast::Ty,
+                          generics: &ast::Generics,
+                          vis: ast::Visibility,
+                          span: Span)
+                          -> Option<String> {
+    let mut result = String::new();
+
+    result.push_str(&format_visibility(vis));
+    result.push_str("type ");
+    result.push_str(&ident.to_string());
+
+    let generics_indent = indent + result.len();
+    let generics_span = mk_sp(span.lo, ty.span.lo);
+    let generics_str = try_opt!(format_generics(context,
```

Comment:
You might be better off using `rewrite_generics` here - there is no where clause so (I think) using `format_generics` is overkill.